rskj-ubuntu-installer/config/regtest.conf

```diff
@@ -231,36 +76,8 @@ wallet {
     accounts = []
 }
 
-wire {
-    protocol: "rsk"
-}
-
 # solc compiler path
-solc.path = <SOLC_PATH>
-
-# not good reputation expiration time in seconds
-scoring {
-    # punishment by node id
-    nodes {
-        # number of nodes to keep scoring
-        number: 100
-        # initial punishment duration (in minutes, default = 10 minutes)
-        duration: 12
-        # punishment duration increment (in percentage, default = 10)
-        increment: 10
-        # maximum punishment duration (in minutes, default = 0 minutes, no maximum)
-        maximum: 0
-    }
-    # punishment by address
-    addresses {
-        # initial punishment duration (in minutes, default = 10 minutes)
-        duration: 12
-        # punishment duration increment (in percentage, default = 10)
-        increment: 10
-        # maximum punishment duration (in minutes, default = 1 week)
-        maximum: 6000
-    }
-}
+solc.path = <SOLC_PATH> 
```

Comment:
The idea is that only `regtest` has `solc.path` enabled, right? Great idea.

Reply:
Yes.